controller/features.go

```diff
@@ -2,29 +2,103 @@ package controller
 
 import (
 	"fmt"
+	jwtgo "github.com/dgrijalva/jwt-go"
 	"github.com/fabric8-services/fabric8-toggles-service/app"
+	"github.com/fabric8-services/fabric8-toggles-service/configuration"
+	"github.com/fabric8-services/fabric8-toggles-service/featuretoggles"
+	"github.com/fabric8-services/fabric8-wit/log"
 	"github.com/goadesign/goa"
+	goajwt "github.com/goadesign/goa/middleware/security/jwt"
 	uuid "github.com/satori/go.uuid"
+	"strings"
 )
 
 // FeaturesController implements the features resource.
 type FeaturesController struct {
```

Comment:
we should probably use a single controller for all requests related to features, but that can be changed in another PR